### PR TITLE
Add support for dockerfile frontend `add-hosts` option

### PR DESCRIPTION
**Pull request by bradfordboyle**

### CODE REVIEW

**Comment by bradfordboyle** [comment]
This is a first pass at addressing #23. Still need to figure out how to run tests.

File: types.go [hunk 1]
```diff
@@ -59,6 +59,8 @@ type Config struct {
 	//
 	// Theoretically this would go away if/when we standardize on OCI.
 	UnpackRootfs bool `json:"unpack_rootfs" envconfig:"optional"`
+
+	AddHosts string `json:"add_hosts" envconfig:"ADD_HOSTS,optional"`
```
Comment (vito) [inline types.go:63]: Since this is a Buildkit-specific feature, can this be called `BUILDKIT_ADD_HOSTS`?

**Review by vito — CHANGES_REQUESTED** [review]
Thanks for working on this! Sorry for the slow turnaround. If you're still interested in working on this, there are better [contributing docs](https://github.com/vito/oci-build-task/blob/master/CONTRIBUTING.md) now, and tests auto-run for PRs now too.

**Review by vito — APPROVED** [review]
lgtm - thanks!

could use a README update but I can add that myself prior to shipping

**Pull request merged**